### DISCUSSION TITLE
LFS files getting corrupted on Pages

### DISCUSSION THREAD

**Discussion by ChaoticRoman** [community/community#102992] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I have noticed that Pages are corrupting my files from LFS.

This PDF is OK here: https://github.com/ChaoticRoman/ChaoticRoman.github.io/blob/main/fluxgate.pdf

But it gets corrupted here: https://romanpavelka.cz/fluxgate.pdf

The file served is in fact text file with content below
```
version https://git-lfs.github.com/spec/v1
oid sha256:2928552761319a7aef3b98351ac9522a28130fc3f0f5f9ce057d199daa48708c
size 4345988
```

I guess LFS support broke...

**Comment by yoannchaudet** [comment]
GitHub Pages does not support LFS and never has.

You can serve content that is stored in LFS in your repo over GitHub Pages, but for that you will need to use a [custom workflow](https://docs.github.com/en/pages/getting-started-with-github-pages/using-custom-workflows-with-github-pages) and make sure that in your `actions/checkout` step, you enable LFS.

**Comment by ChaoticRoman** [comment]
Thank you for your answer. I am pretty sure it worked before and I moved PDFs to LFS long time ago, but it would be not a first time I was wrong. If this is so, let us close this.
